mvm/src/io/context.rs

```diff
@@ -23,13 +21,13 @@ impl ExecutionContext {
     }
 
     pub fn resolve(&self, tag: &StructTag) -> Option<Vec<u8>> {
-        if tag.module.as_str() == TIMESTAMP_MODULE && tag.name.as_str() == CURRENT_TIME_MICROSECONDS {
-             bcs::to_bytes(&self.timestamp).ok()
+        if tag.module.as_str() == TIMESTAMP_MODULE && tag.name.as_str() == CURRENT_TIME_MICROSECONDS
+        {
```

Comment:
nit: Is it rustfmt style? First time I see this

Reply:
Yap. fmt inspection passed.